package.json

```diff
@@ -13,6 +19,7 @@
 		"test:e2e": "playwright test",
 		"prepare": "husky install",
 		"prettier": "prettier \"**/*.+(js|json|ts|jsx|tsx)\"",
+		"prettier-check": "prettier --check \"**/*.+(js|json|ts|jsx|tsx)\"",
```

Comment:
I do not understand how you add the prettier-check library but it doesnt have been formatting